target/linux/ath79/image/generic.mk

```diff
@@ -75,6 +75,17 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/airtight_c-65
+  ATH_SOC := qca9558
+  DEVICE_VENDOR := Airtight
+  DEVICE_MODEL := C-65
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x
+  IMAGE/default := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE)
```

Comment:
Why is this defined when not used/called anywhere?

Reply:
Removed